README.md

```diff
@@ -141,15 +141,16 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [dart](https://github.com/UserNobody14/tree-sitter-dart) (maintained by @Akin909)
 - [ ] [elm](https://github.com/razzeee/tree-sitter-elm)
 - [x] [fennel](https://github.com/travonted/tree-sitter-fennel) (maintained by @TravonteD)
-- [x] [graphql](https://github.com/bkegley/tree-sitter-graphql) (maintained by @bkegley)
 - [x] [go](https://github.com/tree-sitter/tree-sitter-go) (maintained by @theHamsta, @WinWisely268)
+- [x] [graphql](https://github.com/bkegley/tree-sitter-graphql) (maintained by @bkegley)
```

Comment:
why is there a diff here ^^' ?

Reply:
It's on all PRs merged after the graphql PR. It's from the GH actions bot. Needs a rebase.